TODO Api/app/routes/router.py

```diff
@@ -0,0 +1,94 @@
+from fastapi import FastAPI, HTTPException, Depends, APIRouter
+from fastapi.responses import JSONResponse
+from fastapi.encoders import jsonable_encoder
+from uuid import UUID
+import app.models.models as models
+from app.database.database import get_db
+from sqlalchemy.orm import Session
+from app.schemas.schemas import Task, TaskDetails, Priority
+from sqlalchemy import desc
+
+router = APIRouter()
+
+
+# ------------------------------ Get Requests------------------------------
+@router.get("/")
+def get_tasks(db: Session = Depends(get_db)):
+    return db.query(models.Tasks).all()
+
+
+@router.get("/priority-tasks/{priority}")
+def get_tasks(priority: Priority, db: Session = Depends(get_db)):
+    return db.query(models.Tasks).filter(models.Tasks.priority == priority).all()
+
+
+@router.get("/check-status/{status}")
+def get_tasks(status: bool, db: Session = Depends(get_db)):
+    return db.query(models.Tasks).filter(models.Tasks.status == status).all()
+
+
+@router.get("/sort-by-date/")
+def get_tasks(db: Session = Depends(get_db)):
+    return db.query(models.Tasks).order_by(desc(models.Tasks.start_date)).all()
+
+
+# ------------------------------ Post Requests------------------------------
+
+
+@router.post("/", status_code=201)
+async def create_task(task: Task, db: Session = Depends(get_db)):
+    try:
+        task_model = models.Tasks(**task.dict())
+
+        db.add(task_model)
+        db.commit()
+        return JSONResponse(content="Successfully created the task")
+    except Exception as e:
```

Comment:
This is really a bad way to handle a exception, read this https://stackoverflow.com/a/10594142/6832201